docs/getting-started.md

```diff
@@ -255,7 +255,7 @@ bin/cmd/draconctl migrations apply \
   --as-k8s-job \
   --image "${CONTAINER_REPO}/draconctl:${CUSTOM_DRACON_VERSION}" \
   --url "postgresql://dracon:dracon@dracon-enrichment-db.dracon.svc.cluster.local?sslmode=disable" \
-  /etc/dracon/migrations/enrichment
+  --migrations-path /etc/dracon/migrations/enrichment
```

Comment:
this is going to be run inside a migration job, why not have it as an env variable inside the container?